api/v1alpha1/backup/backup.go

```diff
@@ -0,0 +1,169 @@
+// Copyright © 2021 Banzai Cloud
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +kubebuilder:object:generate=true
+
+package backup
+
+import (
+	"fmt"
+
+	"emperror.dev/errors"
+	"github.com/mitchellh/mapstructure"
+	v1 "k8s.io/api/core/v1"
+)
+
+type ServiceSpec struct {
+	ChartValues ValueOverrides `json:"chartValues"`
+}
+
+type ValueOverrides struct {
```

Comment:
All these configuration are required? I guess some of these could be `omitempty`

Reply:
yes, I've added `omitempty` on some of them.